api/v1/resourcesetinputprovider_types.go

```diff
@@ -244,6 +260,17 @@ func (in *ResourceSetInputProvider) GetFilterLimit() int {
 	return 100 // default limit
 }
 
+// GetInvolvedObject returns the involved object for the input provider
+// for cache operations.
+func (in *ResourceSetInputProvider) GetInvolvedObject() cache.InvolvedObject {
```

Comment:
Please move this out of the API package, we shouldn’t have here any deps beyond Kubernetes

Reply:
Fixed, thanks!